lib/ui/recommendation.js

```diff
@@ -81,7 +101,7 @@ Recommendation.prototype = {
     if (this.el) {
       const container = this.win.document.getElementById(RECOMMENDATION_ID);
       this.timeout = this.win.setTimeout(function() {
-        // this.el.collapsed = true;
+        this.el.collapsed = true;
```

Comment:
Yeah, it was. If you're pinning the popup, this hides the recommendation after 50ms, which sort of makes the pin useless. I'll make sure to put this behind a conditional when working on #16